src/main/java/seedu/mtracker/MTracker.java

```diff
@@ -39,7 +39,7 @@ public void run() {
         String[] commandComponents;
 
         do {
-            userInput = parser.getUserInput(MAIN_WORKSPACE);
+            userInput = InputParser.getUserInput(MAIN_WORKSPACE);
```

Comment:
is this supposed to be InputParser or parser

Reply:
InputParser cause getUserInput is static